app/modules/job_control/__init__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+job control module
+============
+"""
```

Comment:
Not sure if I like **job_control** vs something like **ia_job** or **sage_job**. @naknomum ?

Reply:
Could we potentially ever have a job in something other than IA/Sage. As there is nothing Sage specific in here, I chose not to add the Sage name.

Reply:
in old-world we have a couple other things that _might_ be considered joblike; but i dont know they are central enough to distract from this.  (thinking stuff like youtube bot and twitter.  we also had some naming weirdness for IA Task vs Import Task ... but that seems to be a non-issue in nextgen.)

sorta feel like this is enough of our focus (e.g. "_the_ pipeline") that the "jobs" we care most about are likely going to be this stuff.   not super bothered by the genericness here.  maybe just hoping we _wont_ have anything to compete with this in the job/task space?   :crossed_fingers:

Reply:
@naknomum does this mean that you want me to change the name to contain something sage specific?
I have to admit that I really don't like that. 
There is nothing specific in the implementation to Sage and it was really easy to make it so.
It seems an odd design to name a block of code specifically for what it is used to talk to (but doesn't actually talk to) as opposed to what it does.

Reply:
no, @hwindsor - i meant the opposite; sorry.  i meant that i really would like to imagine when we talk of "jobs" in next gen, we _very likely_ will always mean sage/IA/wbia jobs -- so the name here is clear enough.   we may indeed end up with other things in queues but i cant imagine them being as central to functionality as these.